fresque/gitview.py

```diff
@@ -219,3 +219,29 @@ def view_raw_file(repo, identifier, filename):
         headers['Content-Encoding'] = encoding
 
     return (data, 200, headers)
+
+
+@APP.route('/git/<repo>/tags/')
+@APP.route('/git/<repo>/tags')
+def view_tags(repo):
```

Comment:
@pypingou I haven't seen in any of fedora git's repository. I think its not needed but still I don't know why?